Sources/VerticaNetworkHandler/NetworkHandler.swift

```diff
@@ -22,7 +22,8 @@ public struct NetworkHandler {
         decodingStrategy: JSONDecoder.KeyDecodingStrategy? = nil,
         dateDecodingStrategy: JSONDecoder.DateDecodingStrategy? = nil,
         defaultLoggingEnabled: Bool = true,
-        ignoreLoggingForStatusCodes: [Int] = []
+        ignoreLoggingForStatusCodes: [Int] = [],
+        networkIssueRetryCount: Int = 0
```

Comment:
Måske bare? 
```suggestion
        retryCount: Int = 0
```
Jeg tænker dens funktion er ret klar i denne kontekst, og jeg vil nok finde den korte version endnu nemmere at læse.

Reply:
Jo det kunne måske godt give mening, men den retry'er kun når det er et netværksproblem. 
For at undgå at backend fejl (500) bliver forsøgt igen. Så tænkte det var en smule mere tydeligt på denne måde. Men har ikke nogle preferencer i forhold til det 🤷🏼‍♂️

Reply:
Ah, ja, ok. Godt spørgsmål. Empirisk kan jeg dog så sige, at selv med det lange navn forstod jeg ikke helt dens virkemåde 😂